src/utils.ts

```diff
@@ -19,3 +19,16 @@ export function getPagination(
     perPage,
   };
 }
+
+/**
+ * validates if at least one filter is present
+ * @param filters
+ * @param properties
+ */
+export function collectionFilterValidation(filters: any, properties: string) {
```

Comment:
can we use [this type](https://github.com/Pocket/collection-api/blob/main/src/database/types.ts#L179) here for `filters`? speaking of, that type should probably be updated to include `labelExternalIds`, right?

Reply:
@jpetto I've used `any` so that we could technically pass any filter type

Reply:
@jpetto The filter type is already updated in this PR

Reply:
d'oh - i was looking at a single commit last time and not the full change set. _sigh_

carry on, ignore me.